libs/base/Control/Applicative/Const.idr

```diff
@@ -0,0 +1,112 @@
+module Control.Applicative.Const
+
+import Data.Contravariant
+import Data.Bits
+
+public export
+record Const (a : Type) (b : Type) where
+  constructor MkConst
+  value : a
```

Comment:
Can we call this projection `runConst` to match the overall convention?

Reply:
Done.